### ISSUE TITLE
[$15][MSFT-72] [Edge]The NVDA don't read exactly item name in "Sub community" list dropdown, it read the wrong item name

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2786]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/challenges?
2. Navigate to "Filters" button and click on it
3. Click on "Sub community" dropdown and mouse hover on any item in list 

**Expected Result**
The NVDA should read exactly item name in "Sub community" list dropdown

**Actual Result**
The NVDA don't read exactly item name in "Sub community" list dropdown, it read the wrong item name

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F9

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096068 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096068 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096068 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/topcoder-platform/community-app/pull/2952

**Comment by PrakashDurlabhji** [comment]
@gurmeetb new PR https://github.com/topcoder-platform/community-app/pull/2965

**Comment by gurmeetb** [comment]
@PrakashDurlabhji the tests are still failing. Please make sure you mark `tcx_ReadyForReview` only when tests and lint issues pass as mentioned in Bug Bash rule - https://github.com/topcoder-platform/community-app/issues/2915

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097311 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097311 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL: https://test-community-app.topcoder-dev.com/challenges

**Comment by tosha5252** [comment]
Verified Working

OS: Windows 10
Browsers: Chrome and Edge

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097311<br/><br/>```This is an automated message for crazyk via Topcoder X```
